src/frontends/onnx/frontend/src/op/blackmanwindow.cpp

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "op/blackmanwindow.hpp"
+
+#include <memory>
+
+#include "default_opset.hpp"
+#include "utils/common.hpp"
+#include <math.h>
+#define _USE_MATH_DEFINES
+
+OPENVINO_SUPPRESS_DEPRECATED_START
+namespace ngraph {
+namespace onnx_import {
+namespace op {
+namespace set_17 {
+OutputVector blackmanwindow(const Node& node) {
+    const auto size = node.get_ng_inputs().at(0);
+    const auto output_datatype = common::get_ngraph_element_type(node.get_attribute_value("output_datatype", 1));
```

Comment:
```suggestion
    const auto output_datatype = common::get_ngraph_element_type(node.get_attribute_value<int64_t>("output_datatype", 1));
```
I'm able to fix linkage issues on Linux by adding <int64_t> for get_attribute_value in each file, could you do the same, please?